remote-api/src/main/java/org/alfresco/rest/api/impl/TagsImpl.java

```diff
@@ -124,9 +126,19 @@ public int size()
             {
 		        List<Pair<String, NodeRef>> tagNodeRefs = taggingService.addTags(nodeRef, tagValues);
 		        List<Tag> ret = new ArrayList<Tag>(tags.size());
+				tagsByCount = taggingService.findTaggedNodesAndCountByTagName(nodeRef.getStoreRef());
```

Comment:
Is this the only method to find the count for tags? I don't think it returns the full list of tags if there are too many in the system.

Reply:
done